### ISSUE TITLE
Get-AzPolicyStateSummary does not return compliant resource count, only noncompliant ones

### ISSUE THREAD

**Issue by bsod99** [Azure/azure-powershell#16286]
Per the [docs](https://docs.microsoft.com/en-us/powershell/module/az.policyinsights/get-azpolicystatesummary?view=azps-6.5.0#description) this is by design, but why?

For metric/KPI purposes it is required to know both compliant and noncompliant totals for a given assignment. The equivalent azcli command (az policy state summarize) returns both values, but PowerShell does not.

**Comment by dingmeng-xue** [comment]
Policy Insights team, please look into this feature question.

**Comment by kenieva** [comment]
Investigating...

**Comment by navba-MSFT** [comment]
@evanrappe Apologies for the late reply. Could you please use the [Get-AzPolicyState](https://docs.microsoft.com/en-us/powershell/module/az.policyinsights/get-azpolicystate?view=azps-7.2.0) cmdlet and test if that serves your requirement ?

>Get-AzPolicyState -Filter "ComplianceState eq 'Compliant'"

Awaiting your reply.

**Comment by evanrappe** [comment]
@navba-MSFT I think with a bit of local filtering and counting I could get to where we needed for this, but it feels inefficient. It would be great if `Get-AzPolicyStateSummary` reported both compliant/noncompliant counts for a given assignment ID like `az policy state summarize`

Am I missing something, are these two not supposed to be equivalent commands?

**Comment by kenieva** [comment]
@evanrappe At what scope are you making this request? and its just for one particular assignment ID? I want to see if I can reproduce the issue on my side.

**Comment by evanrappe** [comment]
@kenieva Yeah, it's just for one assignment ID, but I'm making this request on a management group above all subscriptions, so there's unfortunately far more than 1000 results.

**Comment by kenieva** [comment]
Can you post the full query you are using? (expect for any MGNames, or other customer data).

**Comment by evanrappe** [comment]
To see that `Get-AzPolicyState` was capped at 1000 results, I used: 
```
(Get-AzPolicyState -ManagementGroupName MGname -Filter "(policyAssignmentId eq '/providers/Microsoft.Management/managementGroups/MGname/providers/Microsoft.Authorization/policyAssignments/policyAssignmentID')").count
```

To see that `Get-AzPolicyStateSummary isn't returning the noncompliant count like az policy state summarize does, I used:
```
get-azpolicystatesummary -managementgroupname "MGname" -filter "(policyAssignmentId eq '/providers/Microsoft.Management/managementGroups/MGname/providers/Microsoft.Authorization/policyAssignments/policyAssignmentID')"
```

**Comment by dunnryan** [comment]
Added the count for the compliant resources to Get-AzPolicyStateSummary. It will be released in the next breaking changes release.
